src/hu_nmt/test/subject_object_augmentator_test.py

```diff
@@ -38,31 +38,111 @@ def test_is_eligible_for_augmentation_and_find_augmentable_candidates(self):
         hun_dep_trees = [self.hungarian_dep_parser.sentence_to_dep_parse_tree(sent_pair[1]) for sent_pair in sentence_pairs]
         eng_graph_wrappers = [DependencyGraphWrapper(tree) for tree in eng_dep_trees]
         hun_graph_wrappers = [DependencyGraphWrapper(tree) for tree in hun_dep_trees]
-        augmentator = SubjectObjectAugmentator(eng_graph_wrappers, hun_graph_wrappers, 0.5, 123, '', 'tsv')
+        augmentator = SubjectObjectAugmentator(eng_graph_wrappers, hun_graph_wrappers, 0.5, 123, [], '', 'tsv')
 
         # assert
         assert_values = [False, False, False, False, False, True]
         for value, hun_g, eng_g in zip(assert_values, hun_graph_wrappers, eng_graph_wrappers):
-            self.assertEqual(value, augmentator.is_eligible_for_augmentation(hun_g, eng_g))
+            self.assertEqual(value, augmentator.is_eligible_for_both_augmentation(hun_g, eng_g))
 
-        augmentator._augmentation_candidate_translations = augmentator.find_augmentable_candidates(augmentator._hun_graphs, augmentator._eng_graphs)
-        self.assertEqual(2, len(augmentator._augmentation_candidate_translations))
+        augmentator._candidate_translations = augmentator.find_candidates(augmentator._hun_graphs, augmentator._eng_graphs)
+        self.assertEqual(2, len(augmentator._candidate_translations['both']))
+
+    def test_is_eligible_for_separate_augmentation_and_find_augmentable_candidates(self):
```

Comment:
Can we split this unit test to two separate ones? The general principle is that a unit test should only test one functionality